core/amber/src/main/resources/python_udf/demo_filter_udf.py

```diff
@@ -0,0 +1,2 @@
+def filter_function(row, args):
```

Comment:
please include some comment and explain the input and output of this function with an example.

Reply:
OK, will do.